src/translator/byte_array_util.cpp

```diff
@@ -0,0 +1,34 @@
+#include "byte_array_util.h"
+#include <stdlib.h>
+#include <iostream>
+
+namespace marian {
+namespace bergamot {
+
+AlignedMemory loadFileToMemory(const std::string& path, bool isModelFile){
+  uint64_t fileSize = filesystem::fileSize(path);
+  io::InputFileStream in(path);
+  ABORT_IF(in.bad(), "Failed opening file stream: {}", path);
+  size_t alignment = isModelFile? 256 : 64;
```

Comment:
Can't this be accepted as an argument (alignment) instead of isModelFile and generalize this function a bit more? This doesn't do anything specifically to model does it, it only switches alignment, which can be 

```cpp
AlignedMemory loadFileToMemory(const std::string& path, size_t alignment)
```